src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1913,11 +1913,11 @@ void func_80832E48(Player* this, s32 flags) {
     func_808322FC(this);
 }
 
-void func_80832F54(PlayState* play, Player* this, s32 flags) {
+void Player_AnimReplaceApplyFlags(PlayState* play, Player* this, s32 flags) {
```

Comment:
Is this suppose to mean "replace the `applyFlags`", or "replace and apply these `flags`"?

Reply:
it means ' in the "AnimReplace" subsystem, "Apply" (an anim replacement) with "Flags" '

But I can see how it's a weird name.
Note this does not change the animation, it's instead typically called after an animation is changed

See also the "Player_AnimReplace... functions" bit in my opening comment above
Basically "AnimReplace" as in it does a bit more than changing the animation (the functions wrapping this one, at least)

As fig said the names are very literal :sweat: 
The flags here are mostly SkelAnime.moveFlags (see #1489 ) so that could be worth to keep in mind naming this. (but also I'm not sure `SkelAnime.moveFlags` is a perfect name so...)

Reply:
I added a comment, let me know if it helps (not the greatest comment)
https://github.com/zeldaret/oot/pull/1437/commits/772f5556b02e658295e6f066feefae56e7155ae1